src/components/select/select.ts

```diff
@@ -26,6 +26,7 @@ import type SlRemoveEvent from '../../events/sl-remove';
  * @documentation https://shoelace.style/components/select
  * @status stable
  * @since 2.0
+ * @pattern stable
```

Comment:
Is this also supposed to have a `@figma` annotation?

Reply:
When there's no annotation, it defaults to just showing the "Figma Needed" tag, which is right for `Select` -- right now we have no Figma component for Select.